WebApplication1/Dto/UpdateMovieDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace WebApplication1.Dto
+{
+    public class UpdateMovieDto
+    {
+        public string Name { get; set; }
```

Comment:
you need to also have the id of the item you are going to update, because if you did not it will instead create a new item with the same name but a different ID